tests/test_modeling_common.py

```diff
@@ -3720,16 +3730,33 @@ def get_mean_reldiff(failcase, x, ref, atol, rtol):
                                         outputs_eager = model_eager(**prepared_inputs)
                                         outputs_sdpa = model_sdpa(**prepared_inputs)
 
-                                logits_eager = (
-                                    outputs_eager.hidden_states[-1]
-                                    if not is_encoder_decoder
-                                    else outputs_eager.decoder_hidden_states[-1]
-                                )
-                                logits_sdpa = (
-                                    outputs_sdpa.hidden_states[-1]
-                                    if not is_encoder_decoder
-                                    else outputs_sdpa.decoder_hidden_states[-1]
-                                )
+                                # `CLIPOutput` doesn't have "hidden_states" or "decoder_hidden_states".
+                                if model_class.__name__ == "CLIPModel":
+                                    logits_eager = torch.cat(
+                                        [outputs_eager.image_embeds, outputs_eager.text_embeds], dim=0
+                                    )
+                                else:
+                                    logits_eager = (
+                                        outputs_eager.hidden_states[-1]
+                                        if not is_encoder_decoder
+                                        else outputs_eager.decoder_hidden_states[-1]
+                                    )
```

Comment:
To be consistent with the if/else structure here

```suggestion
                                elif is_encoder_decoder:
                                    logits_eager = outputs_eager.decoder_hidden_states[-1]
                                else:
                                    logits_eager = outputs_eager.hidden_states[-1]
```

Reply:
Done in 773a0a370.